packages/angular-test-app/src/preview-examples/blind-variants.ts

```diff
@@ -11,6 +11,7 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-example',
-  templateUrl: './blind.html',
+  styleUrls: [`../../../documentation/static/styles/blind.css`],
```

Comment:
Do not reference a other project via relative path. We done this one in tsconfig. Its a pain also to run die angular-test-app standalone outside of the mono repo.

Also please exaplain whats the idea behind reference something from a another package?